packages/web-components/fast-foundation/src/flipper/flipper.template.ts

```diff
@@ -17,26 +38,7 @@ export function flipperTemplate(
             class="${x => x.direction} ${x => (x.disabled ? "disabled" : "")}"
             @keyup="${(x, c) => x.keyupHandler(c.event as KeyboardEvent)}"
         >
-            ${when(
-                x => x.direction === FlipperDirection.next,
-                html`
-                    <span part="next" class="next">
-                        <slot name="next">
-                            ${options.next || ""}
-                        </slot>
-                    </span>
-                `
-            )}
-            ${when(
-                x => x.direction === FlipperDirection.previous,
-                html`
-                    <span part="previous" class="previous">
-                        <slot name="previous">
-                            ${options.previous || ""}
-                        </slot>
-                    </span>
-                `
-            )}
+            ${x => setFlipperTemplateByDirection(x.direction, options)}
```

Comment:
Do we need a when directive at all? Would 
```ts
                    html`<span part="${x => x.direction}" class="${x => x.direction}">
                        <slot name="${x => x.direction}">
                            ${x => options[x.direction] || ""}
                        </slot>
                    </span>`
```
suffice?

Reply:
We probably *don't* need a class on the span even - CSS can target the part attribute.

Reply:
I had something similar, but part of me wonders if we need to bind direction across all of these instances or if it would be more efficient to grab it once and set it for everything within this portion of the template.

Reply:
> We probably _don't_ need a class on the span even - CSS can target the part attribute.

I think for convention here I'll leave that - if we want to move to targeting parts within the component stylesheet I think we can take that up as a pattern...but the delta (as you noted offline) is probably pretty negligible.